process/factory/shard/vmContainerFactory_test.go

```diff
@@ -137,6 +139,33 @@ func TestNewVMContainerFactory_NilHasherShouldErr(t *testing.T) {
 	assert.Equal(t, process.ErrNilHasher, err)
 }
 
+func TestNewVMContainerFactory_NilPubKeyConverterShouldErr(t *testing.T) {
```

Comment:
Perhaps we could add a working test to test that the decoded addresses are correctly saved internally in that map?

Reply:
ok. doing it.